wandering-warriors/main.py

```diff
@@ -1,30 +1,54 @@
 from kivy.app import App
-from kivy.uix.widget import Widget
 from kivy.lang import Builder
+from kivy.uix.screenmanager import ScreenManager, Screen
+from kivy.uix.widget import Widget
 
 
-class ToolManager(Widget):
+class Calculator(Screen):
     pass
 
 
-class Abacus(Widget):
+class Settings(Screen):
     pass
 
 
-class CuneiformDrawingInput(Widget):
+class TopMenu(Widget):
+    pass
+
+
+class Abacus(Widget):
     pass
 
 
 class Ledger(Widget):
     pass
 
 
-class AbacusApp(App):
+class OperationsBar(Widget):
+    pass
+
+
+class CuneiformDrawingInput(Widget):
+    pass
+
+
+class Screen:
+    # There will be three screens: title, settings, board.
+    # A class that keeps track of gamestates will likely be needed.
+
+    def __init__(self):
+        self.sm = ScreenManager()
+        self.sm.add_widget(Calculator(name='calculator'))
+
+    def get_manager(self):
+        return self.sm
+
+
+class Application(App):
     def build(self):
-        Builder.load_file("abacus.kv")
-        app = ToolManager()
-        return app
+        Builder.load_file("app.kv")
```

Comment:
Should we follow kivy conventions, call this class `<something>App`, and use `<something>.kv`? This may make people know where the config is stored without having to look here? At the same time, it's the only `.kv` file in the directory so it should be fine...

Reply:
Sure I can do that

Reply:
Sure I can do that